plugins/CoreAdminHome/Tasks.php

```diff
@@ -112,8 +112,12 @@ public function invalidateOutdatedArchives()
             return;
         }
 
-        $cronArchive = new CronArchive();
-        $cronArchive->invalidateArchivedReportsForSitesThatNeedToBeArchivedAgain();
+        $idSites = Request::processRequest('SitesManager.getAllSites');
```

Comment:
just fyi @diosmosis could have used `getAllSitesId` method but not too important. Won't need to change it

Reply:
actually quickly changed it just fyi